ci-operator/config/openshift/kube-proxy/OWNERS

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT; this file is auto-generated using tools/populate-owners.
+# from https://github.com/openshift/kube-rbac-proxy/blob/96e27d5465ef09c942a76bec041df1eec2080179/OWNERS
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
```

Comment:
@pecameron should we add some people from networking to this list as well?

Reply:
The comment above says "DO NOT EDIT". I will look at generating the OWNERS file...

Reply:
Ah, ok.